app/pages/project/vpcs/InternetGatewayPage/InternetGatewayPage.tsx

```diff
@@ -0,0 +1,119 @@
+
+import { Networking16Icon, Networking24Icon } from '@oxide/design-system/icons/react'
+
+import { apiQueryClient, usePrefetchedApiQuery } from '~/api'
+import { DocsPopover } from '~/components/DocsPopover'
+import { MoreActionsMenu } from '~/components/MoreActionsMenu'
+import { RouteTabs, Tab } from '~/components/RouteTabs'
+import { getInternetGatewaySelector, useInternetGatewaySelector } from '~/hooks/use-params'
+import { DescriptionCell } from '~/table/cells/DescriptionCell'
+import { DateTime } from '~/ui/lib/DateTime'
+import { Message } from '~/ui/lib/Message'
+import { PageHeader, PageTitle } from '~/ui/lib/PageHeader'
+import { PropertiesTable } from '~/ui/lib/PropertiesTable'
+import { Truncate } from '~/ui/lib/Truncate'
+import { ALL_ISH } from '~/util/consts'
+import { docLinks } from '~/util/links'
+import { pb } from '~/util/path-builder'
+
+InternetGatewayPage.loader = async function ({ params }: LoaderFunctionArgs) {
+  console.log('InternetGatewayPage.loader')
+  const { project, vpc, gateway } = getInternetGatewaySelector(params)
+  console.log({ project, vpc, gateway })
+  const query = { project, vpc, gateway, limit: ALL_ISH }
+  await Promise.all([
+    apiQueryClient.prefetchQuery('internetGatewayView', {
+      query: { project, vpc },
+      path: { gateway },
+    }),
+    apiQueryClient.prefetchQuery('internetGatewayIpAddressList', { query }),
+    apiQueryClient.prefetchQuery('internetGatewayIpPoolList', { query }),
+  ])
+  return null
+}
+
+export function InternetGatewayPage() {
+  const gatewaySelector = useInternetGatewaySelector()
+  const { project, vpc, gateway } = gatewaySelector
+  const {
+    data: { id, description, name, timeCreated, timeModified },
+  } = usePrefetchedApiQuery('internetGatewayView', {
+    query: { project, vpc },
+    path: { gateway },
+  })
+
+  const actions = useMemo(
+    () => [
+      {
+        label: 'Copy ID',
+        onActivate() {
+          window.navigator.clipboard.writeText(id || '')
+        },
+      },
+    ],
+    [id]
+  )
+
+  return (
+    <>
+      <PageHeader>
+        <PageTitle icon={<Networking24Icon />}>{name}</PageTitle>
+        <div className="inline-flex gap-2">
+          <DocsPopover
+            heading="Internet Gateways"
+            icon={<Networking16Icon />}
+            summary="Internet gateways … 🚨🙈🚨🙉🚨🙊🚨 … just using emojis here so we spot it more easily in the PR; this copy needs eyes 👀"
+            links={[docLinks.vpcs, docLinks.firewallRules]}
+          />
+          <MoreActionsMenu label="VPC actions" actions={actions} />
+        </div>
+      </PageHeader>
+      <PropertiesTable.Group className="mb-8">
+        <PropertiesTable>
+          <PropertiesTable.Row label="Description">
+            <DescriptionCell text={description} />
+          </PropertiesTable.Row>
+          <PropertiesTable.Row label="ID">
+            <Truncate text={id} maxLength={32} hasCopyButton />
+          </PropertiesTable.Row>
```

Comment:
This isn't necessary, as we have the `MoreActionsMenu` button in the header with the `Copy ID` button, but I didn't see a clear "fourth" aspect to the internet gateway metadata to highlight. If we do keep this, we can drop the `MoreActionsMenu`, as it's redundant. I suspect dropping this one is the better approach, though, if we can figure out what to put here in its place.